Improve wording in Getting Started guide (#4369)
elixir-ecto/ecto: guides/introduction/Getting Started.md

This module is what we'll be using to query our database shortly. It uses the `Ecto.Repo` module, and the `otp_app` tells Ecto which Elixir application it can look for database configuration in. In this case, we've specified that it is the `:friends` application where Ecto can find that configuration and so Ecto will use the configuration that was set up in `config/config.exs`. Finally, we configure the database `:adapter` to Postgres.

> The `Friends.Repo` must be setup as a supervisor within the application's supervision tree, which we can do in `lib/friends/application.ex`, inside the `start/2` function:

Reviewer: ```suggestion
Finally, the `Friends.Repo` must be started within the application's supervision tree, which we can do in `lib/friends/application.ex`, inside the `start/2` function:
```
